src/nvim/highlight_group.c

```diff
@@ -135,263 +135,321 @@ static const char e_missing_argument_str[]
+  "default link DiagnosticUnnecessary      Comment",
+
   // Text
-  "default link @text.literal Comment",
+  "default link @text.literal   Comment",
   "default link @text.reference Identifier",
-  "default link @text.title Title",
-  "default link @text.uri Underlined",
+  "default link @text.title     Title",
+  "default link @text.uri       Underlined",
   "default link @text.underline Underlined",
-  "default link @text.todo Todo",
+  "default link @text.todo      Todo",
 
   // Miscs
-  "default link @comment Comment",
+  "default link @comment     Comment",
   "default link @punctuation Delimiter",
 
   // Constants
-  "default link @constant Constant",
-  "default link @constant.builtin Special",
-  "default link @constant.macro Define",
-  "default link @define Define",
-  "default link @macro Macro",
-  "default link @string String",
-  "default link @string.escape SpecialChar",
-  "default link @string.special SpecialChar",
-  "default link @character Character",
+  "default link @constant          Constant",
+  "default link @constant.builtin  Special",
+  "default link @constant.macro    Define",
+  "default link @define            Define",
+  "default link @macro             Macro",
+  "default link @string            String",
+  "default link @string.escape     SpecialChar",
+  "default link @string.special    SpecialChar",
+  "default link @character         Character",
   "default link @character.special SpecialChar",
-  "default link @number Number",
-  "default link @boolean Boolean",
-  "default link @float Float",
+  "default link @number            Number",
+  "default link @boolean           Boolean",
+  "default link @float             Float",
 
   // Functions
-  "default link @function Function",
+  "default link @function         Function",
   "default link @function.builtin Special",
-  "default link @function.macro Macro",
-  "default link @parameter Identifier",
-  "default link @method Function",
-  "default link @field Identifier",
-  "default link @property Identifier",
-  "default link @constructor Special",
+  "default link @function.macro   Macro",
+  "default link @parameter        Identifier",
+  "default link @method           Function",
+  "default link @field            Identifier",
+  "default link @property         Identifier",
+  "default link @constructor      Special",
 
   // Keywords
   "default link @conditional Conditional",
-  "default link @repeat Repeat",
-  "default link @label Label",
-  "default link @operator Operator",
-  "default link @keyword Keyword",
-  "default link @exception Exception",
-
-  "default link @variable Identifier",
-  "default link @type Type",
+  "default link @repeat      Repeat",
+  "default link @label       Label",
+  "default link @operator    Operator",
+  "default link @keyword     Keyword",
+  "default link @exception   Exception",
+
+  "default link @variable        NONE",  // don't highlight to reduce visual overload
```

Comment:
Just updated to nvim 0.10, the biggest visual change is that all the variables have changed colors with a custom colorscheme, this strong contrast with the context seems to be more "visual overload" than before, I would suggest to keep linking to `Identifier`. Thanks!

Reply:
This is not up for debate; the colorscheme is final. Feel free to customize it yourself, of course.